torch_geometric/data/temporal.py

```diff
@@ -212,6 +212,10 @@ def num_edges(self) -> int:
         r"""Alias for :meth:`~torch_geometric.data.TemporalData.num_events`."""
         return self.num_events
 
+    @property
+    def edge_index(self) -> Any:
+        return self['edge_index'] if 'edge_index' in self._store else None
```

Comment:
`edge_index` can be also obtained by stacking `src` and `dst`, WDYT?

Reply:
This is in order to build a graph based on temporal edges by creating a TemporalData that networkx can understand just like with the Data object. I agree, one can stack `src` and `dst' but this is just a bit cleaner.

Reply:
Yep, I mean we can do something like:
```python
    @property
    def edge_index(self):
        edge_index = self['edge_index'] 
        if edge_index is None and not (self.src is None or self.dst is None):
            edge_index = torch.stack([self.src, self.dst], dim=0)
        return edge_index
```